src/perf/dbserver/.gitignore

```diff
@@ -0,0 +1,360 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
```

Comment:
Why is this needed?

Reply:
Theres a lot of extra ignores for C# projects, especially ones published to azure. I can move most of these to the root gitignore, but usually I see subprojects like this using their own in case they want to get split out.

Reply:
I thought our top level .gitignore had pretty much all of these. What exactly is missing from it that makes this one necessary?

Reply:
Turns out it does. I thought It was missing the azure ignores. Removed.